plugins/ctf/fs-sink/writer.c

```diff
@@ -335,8 +338,7 @@ enum bt_component_status writer_component_init(
 
 	value = bt_value_map_get(params, "path");
 	if (!value || bt_value_is_null(value) || !bt_value_is_string(value)) {
-		fprintf(writer_component->err,
-				"[error] output path parameter required\n");
+		BT_LOGE_STR("Output path parameter required");
```

Comment:
Indicate which parameter exactly, for example:

    Missing mandatory `path` parameter.